psi_codex/codex_catastrophe.py

```
@@ -4,97 +4,145 @@
+
+def michael_stabilizer(psi: float, eta: float, system_state: dict) -> dict:
+    """
+    Michael (id 50) Stabilizer Logic.
+    Guards Azazel gate and manages Samael/Belial boundaries.
+    """
+    interventions = []
+
+    # 1. Oppose Samael (id 49) if his entropy rises
+    if ENTITIES[49]["eta"] > 0.08:
+        system_state['ethical_tension'] = max(0.0, system_state.get('ethical_tension', 0.0) - 0.1)
+        interventions.append("Michael opposes Samael – ethical tension reduced.")
```

Ruff (0.15.12)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)